crates/swc_html_minifier/src/lib.rs

```diff
@@ -470,351 +471,78 @@ impl Minifier<'_> {
         attribute_name: &str,
         attribute_value: &str,
     ) -> bool {
```

Comment:
The return type could be written as `Option<bool>`, so you could use the `?` in function body.

Reply:
But why? I feel fine with `bool`